hazelcast/test/src/sql_test.cpp

```diff
@@ -221,10 +259,53 @@ namespace hazelcast {
 namespace client {
 namespace test {
 
+template<typename T, typename = void>
+struct generator;
+
+template<typename T>
+struct generator<T, typename std::enable_if<std::is_integral<T>::value>::type>
```

Comment:
I find it better here for now because it is only used by sql_test.cpp. It is obvious that It can be reused in the future but I don't want to move unless it is really needed.